docs/install/_index.md

```diff
@@ -23,9 +23,7 @@ showlandingtoc: "false"
 
 You can install the Serving component, Eventing component, or both on your cluster by using one of the following deployment options:
 
-- Using a YAML-based installation:
-  - [Installing Serving using YAML files](./install-serving-with-yaml)
-  - [Installing Eventing using YAML files](./install-eventing-with-yaml)
+- Using a [YAML-based installation](./prerequisites.md)
```

Comment:
Did @snneji not just intentionally split this?

Reply:
Yes! It's still split, it just starts with the Pre-Reqs then splits off.

Reply:
The https://knative.dev/development/install/prerequisites/ page doesn't link to serving or eventing as next steps or anything though and they're on separate pages, so to me it would make more sense to keep separate links to those two sections, or else add a "Next steps" section that links to those installation pages?